permissions/permissions.go

```diff
@@ -0,0 +1,394 @@
+	mailSettingsForwardSpam      = MailSettings + ".forward_spam"
+	mailSettingsPlainContent     = MailSettings + ".plain_content"
+	mailSettingsSpamCheck        = MailSettings + ".spam_check"
+	mailSettingsTemplate         = MailSettings + ".template"
+	partnerSettingsNewRelic      = PartnerSettings + ".new_relic"
+	partnerSettingsSendWithUs    = PartnerSettings + ".sendwithus"
+	subusersCredits              = Subusers + ".credits"
+	subusersCreditsRemaining     = subusersCredits + ".remaining"
+	subusersMonitors             = Subusers + ".monitor"
+	subusersReputations          = Subusers + ".reputations"
+	subusersStats                = Subusers + ".stats"
+	subusersStatsMonthly         = subusersStats + ".monthly"
+	subusersStatsSums            = subusersStats + ".sums"
+	subusersSummary              = Subusers + ".summary"
+	suppressionBounces           = Suppression + ".bounces"
+	suppressionBlocks            = Suppression + ".blocks"
+	suppressionInvalidEmails     = Suppression + ".invalid_emails"
+	suppressionSpamReports       = Suppression + ".spam_reports"
+	suppressionUnsubscribes      = Suppression + ".unsubscribes"
+	templatesVersions            = Templates + ".versions"
+	templatesVersionsActivate    = templatesVersions + ".activate"
+	trackingSettingsClick        = TrackingSettings + ".click"
+	trackingSettingsGoogle       = TrackingSettings + ".google_analytics"
+	trackingSettingsOpen         = TrackingSettings + ".open"
+	trackingSettingsSubscription = TrackingSettings + ".subscription"
+	user                         = "user"
+	userAccount                  = user + ".account"
+	userCredits                  = user + ".credits"
+	userEmail                    = user + ".email"
+	userMultifactorAuth          = user + ".multifactor_authentication"
+	userPassword                 = user + ".password"
+	userProfile                  = user + ".profile"
+	userSettingsEnforcedTLS      = user + ".settings.enforced_tls"
+	userTimezone                 = user + ".timezone"
+	userUsername                 = user + ".username"
+	userWebhooks                 = user + ".webhooks"
+	userWebhooksEvent            = userWebhooks + ".event"
+	userWebhooksEventSettings    = userWebhooksEvent + ".settings"
+	userWebhooksEventTest        = userWebhooksEvent + ".test"
+	userWebhooksParse            = userWebhooks + ".parse"
+	userWebhooksParseSettings    = userWebhooksParse + ".settings"
+	userWebhooksParseStats       = userWebhooksParse + ".stats"
+	accessSettings               = "access_settings"
+	accessSettingsActivity       = accessSettings + ".activity"
+	accessSettingsWhitelist      = accessSettings + ".whitelist"
+	read                         = "read"
+	update                       = "update"
+	create                       = "create"
+	delete                       = "delete"
+	send                         = "send"
+	delimiter                    = "."
+	readOnlyAccess               = "readOnly"
+	fullAccess                   = "full"
```

Comment:
I believe this was to be a boolean value.

Reply:
Sure

Reply:
I am not sure about the boolean value here. Right now we have a distinction between "readOnly" and "full". in `GetPermissions` method. If we were to change this, we'd need to change the method signature and decide on the name of this boolean flag.